src/main/java/cat/udl/eps/entsoftarch/textannot/repository/AnnotationRepository.java

```diff
@@ -13,6 +14,7 @@
 public interface AnnotationRepository extends PagingAndSortingRepository<Annotation, Integer> {
     Annotation findByid(@Param("id") Integer id);
     List<Annotation> findByAnnotated(@Param("value") Sample value);
```

Comment:
The problem is in the name of the attributes in the class, but the name of the find methods is very bad.
And also  the parameter could get a better name.

If you use HAL to get the methods, you get:

{
  "_links": {
    "findByAnnotated": {
      "href": "http://localhost:8080/annotations/search/findByAnnotated{?value}",
      "templated": true
    },
    "findByid": {
      "href": "http://localhost:8080/annotations/search/findByid{?id}",
      "templated": true
    },
    "findByTagged": {
      "href": "http://localhost:8080/annotations/search/findByTagged{?value}",
      "templated": true
    },
    "self": {
      "href": "http://localhost:8080/annotations/search"
    }
  }
}

BTW: I know  it's not your fault, but has the method 

      Annotation findByid(@Param("id") Integer id); 

any sense?

Please, when adding code to an existing class, try to improve it (or, al least, try to find possible errors).